src/server.c

```diff
@@ -416,7 +344,14 @@ main(int argc, char **argv) {
     lwsl_notice("tty configuration:\n");
     if (server->credential != NULL)
         lwsl_notice("  credential: %s\n", server->credential);
-    lwsl_notice("  start command: %s\n", server->command);
+    
+    lwsl_notice("  start command: \n");
+    for (c = 0; c < argc; c++) {
+        lwsl_notice(" %s\n", argv[c]);
```

Comment:
This isn't aligned with `start command`. Also, I'd like to print the full command line within a single line.